dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/sqlclassification/KyuubiGetSqlClassification.scala

```diff
@@ -35,28 +35,25 @@ import org.apache.kyuubi.sql.KyuubiSQLConf._
  *  If there have no this named jsonFile,
  *  the service will upload the default matching rule: sql-classification-default.json.
  */
-object KyuubiGetSqlClassification {
-
+object KyuubiGetSqlClassification extends Logging {
   private val jsonNode: Option[JsonNode] = {
     SQLConf.get.getConf(SQL_CLASSIFICATION_ENABLED) match {
       case true =>
         val objectMapper = new ObjectMapper
-        var url: URL = Thread.currentThread().getContextClassLoader
-          .getResource("sql-classification.json")
+        var url: URL = getClass.getClassLoader.getResource("sql-classification.json")
         if (url == null) {
-          url = Thread.currentThread().getContextClassLoader
-            .getResource("sql-classification-default.json")
+          logInfo("sql-classification.json is not found, use default config instead")
+          url = getClass.getClassLoader.getResource("sql-classification-default.json")
         }
-        val defaultSqlClassificationFile = url.getPath
-        Some(objectMapper.readTree(new File(defaultSqlClassificationFile)))
+        Some(objectMapper.readTree(url))
```

Comment:
1. Shall we use `Option` instead?
2. Shall we also handle a malformed JSON case? `objectMapper.readTree()` may throws `IOException/JsonProcessingException`.

Reply:
I think it's ok to throw exception if the error config from user. And by default, we will never get malformed data.

Reply:
here should check the jsonFile's format that the user provided... 
i plan to do it after this PR...

Reply:
if we really want to check the file whether is malformed or not, we can do it at a new PR in further.